frontend/components/site/SiteUsers.jsx

```diff
@@ -72,6 +80,9 @@ const SiteUsers = ({ site, user }) => {
           )}
         </tbody>
       </table>
+      <div className="offset-top">
+        {renderUserActions(user.userActions)}
```

Comment:
I think we should we add `userActions` to the `USER` propType in `propTypes.js`:

```js
export const USER = PropTypes.shape({
  id: PropTypes.number,
  email: PropTypes.string,
  username: PropTypes.string,
  userActions: PropTypes.arrayOf(USER_ACTION),
});
```

can do in separate PR though.

Reply:
I didn't know if it should be added since we pass some additional information to the user reducer that isn't enumerated in this propType declaration. Basically I didn't know if we just wanted to validate this subset for some reason